src/data/item-desc.csv

```diff
@@ -1,71 +1,71 @@
+"蹄鉄型の銀色のお守り。\nショップで様々なものと交換できる。","<size=22>A silver amulet in the shape of a horseshoe. Can be exchanged for various items in the shop.</size>"
+"蹄鉄型の金色のお守り。\nショップで様々なものと交換できる。","<size=22>A golden amulet in the shape of a horseshoe. Can be exchanged for various items in the shop.</size>"
+"蹄鉄型の虹色のお守り。\nショップで様々なものと交換できる。","<size=22>A rainbow-colored amulet in the shape of a horseshoe. Can be exchanged for various items in the shop.</size>"
+"幸運を呼ぶ四つ葉のクローバー。\nショップで様々なものと交換できる。","<size=22>A four-leafed clover that brings good luck. Can be exchanged for various items in the shop.</size>"
+"イベントを遊ぶことで入手できる。\n集めることで様々な報酬を獲得できる。","<size=22>Points earned by playing through events. You can earn various awards by collecting these.</size>"
+"備えがあればうれしいな。\nトレセン学園の様々な場面で使用できる。","<size=22>Being well prepared means having no worries in times of need. Can be used in various parts of Tracen Academy, such as shops.</size>"
+"レース前にウマ娘の能力の分析ができる","<size=22>Makes it possible to analyze the abilities of opponents before a race.</size>"
+"ダートGⅠ・フェブラリーSを優勝した証。\n覚醒Lv強化で使用できる。","<size=22>Proof of having won the G1 dirt race February Stakes. Used to raise the awakening level of a character.</size>"
+"短距離GⅠ・高松宮記念を優勝した証。\n覚醒Lv強化で使用できる。","<size=22>Proof of having won the G1 short-distance race Takamatsunomiya Kinen. Used to raise the awakening level of a character.</size>"
+"中距離GⅠ・大阪杯を優勝した証。\n覚醒Lv強化で使用できる。","<size=22>Proof of having won the G1 medium-distance race Osaka Hai. Used to raise the awakening level of a character.</size>"
+"クラシック級GⅠ・桜花賞を優勝した証。\n覚醒Lv強化で使用できる。","<size=22>Proof of having won the Classics G1 race Oka Sho. Used to raise the awakening level of a character.</size>"
+"クラシック級GⅠ・皐月賞を優勝した証。\n覚醒Lv強化で使用できる。","<size=22>Proof of having won the Classics G1 race Satsuki Sho. Used to raise the awakening level of a character.</size>"
+"長距離GⅠ・天皇賞（春）を優勝した証。\n覚醒Lv強化で使用できる。","<size=22>Proof of having won the G1 long-distance race Tenno Sho (Spring). Used to raise the awakening level of a character.</size>"
+"クラシック級GⅠ・NHKマイルCを優勝\nした証。覚醒Lv強化で使用できる。","<size=22>Proof of having won the Classics G1 race NHK Mile Cup. Used to raise the awakening level of a character.</size>"
+"マイルGⅠ・ヴィクトリアマイルを\n優勝した証。覚醒Lv強化で使用できる。","<size=22>Proof of having won the G1 mile race Victoria Mile. Used to raise the awakening level of a character.</size>"
+"クラシック級GⅠ・オークスを優勝した証。\n覚醒Lv強化で使用できる。","<size=22>Proof of having won the Classics G1 race Oaks. Used to raise the awakening level of a character.</size>"
+"クラシック級GⅠ・日本ダービーを\n優勝した証。覚醒Lv強化で使用できる。","<size=22>Proof of having won the Classics G1 race Japan Derby. Used to raise the awakening level of a character.</size>"
+"マイルGⅠ・安田記念を優勝した証。\n覚醒Lv強化で使用できる。","<size=22>Proof of having won the G1 Mile race Yasuka Kinen. Used to raise the awakening level of a character.</size>"
+"中距離GⅠ・宝塚記念を優勝した証。\n覚醒Lv強化で使用できる。","<size=22>Proof of having won the G1 medium-distance race Takarazuka Kinen. Used to raise the awakening level of a character.</size>"
```

Comment:
what do you think if we shorten all the garland items to
"Gx Race name victory proof, used to raise awakening lv." or something like that if anyone can think of a better one

Reply:
Agree; imo the descriptions are all far too long. You can only add a few bit more text with `<size>` markup, and if the number is too small the actual textbox width is shortened.

As a general rule of thumb, the translated text should be about as long (or very slightly longer) than the japanese text (in monospace, as viewed in github).

Reply:
Yeah, I figured the text would be too long. For starters, I translated the descriptions accurately to the original because we can always cut it down. I'm fine with that garland item shortening.